### PR TITLE
mrc-4170 Privacy statement

### PR DESCRIPTION
**Pull request by EmmaLRussell**
This branch adds a Privacy statement, linked from the navbar on the landing page (next to the Accessibility link). The text of the statement is based on the corresponding statement in HINT. 

I've also enabled reloading the Accessibility and Privacy pages by defining those routes in the `HomeController` in the back end. I also noticed that reload on the Strategise page fails - that's going to be trickier to fix, so have created a ticket: https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-4193/

This branch has been deployed to mint-dev, and the privacy notice can be viewed here: https://mint-dev.dide.ic.ac.uk/privacy

### CODE REVIEW

File: src/app/static/src/app/components/privacyPage.vue [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                </p>
+
+                <h2>How long will we use your personal data for?</h2>
+                <ul>
+                    <li>
+                      We will retain your personal data indefinitely to fulfil the purposes we collected it for, including for the purposes
+                      of satisfying any legal, accounting, or reporting requirements.
+                    </li>
+                    <li>We therefore will remove your data when the site is discontinued.</li>
+                    <li>We will remove your data when you request we do so.</li>
```
Comment (richfitz) [inline src/app/static/src/app/components/privacyPage.vue:51]: Practically, how would this work?

File: src/app/static/src/app/components/privacyPage.vue [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                <p>
+                  We have put in place appropriate security measures to prevent your personal data from being accidentally lost, used or
+                  accessed in an unauthorised way, altered or disclosed. In addition, we limit access to your personal data to the
+                  administrators of MINT, who have a business need to know, and are subject to a duty of confidentiality.
+                </p>
+
+                <h2>How long will we use your personal data for?</h2>
+                <ul>
+                    <li>
+                      We will retain your personal data indefinitely to fulfil the purposes we collected it for, including for the purposes
```
Comment (richfitz) [inline src/app/static/src/app/components/privacyPage.vue:47]: do we not rotate the logs pretty quickly?

**Review by richfitz — APPROVED** [review]
LGTM, but two comments for us to ponder

File: src/app/static/src/app/components/privacyPage.vue (hunk 1 above)
Comment (EmmaLRussell) [reply src/app/static/src/app/components/privacyPage.vue:51]: Yeah, good question - they'd have to tell us their IP address I guess :shrug:

File: src/app/static/src/app/components/privacyPage.vue (hunk 2 above)
Comment (EmmaLRussell) [reply src/app/static/src/app/components/privacyPage.vue:47]: Don't think so, we still seem to have some stuff from years ago on there. Maybe we should!

**Pull request merged**